src/Sylius/Bundle/CurrencyBundle/Resources/config/serializer/Model.Currency.yml

```diff
@@ -0,0 +1,41 @@
+Sylius\Component\Currency\Model\Currency:
+    exclusion_policy: ALL
+    xml_root_name: currency
+    properties:
+        id:
+            expose: true
+            type: integer
+            xml_attribute: true
+            groups: [Default, Detailed]
+        code:
+            expose: true
+            type: string
+            groups: [Default, Detailed]
+        exchangeRate:
+            expose: true
+            type: float
+            groups: [Default, Detailed]
+        enabled:
+            expose: true
+            type: boolean
+            groups: [Default, Detailed]
+        base:
+            expose: true
+            type: boolean
+            groups: [Default, Detailed]
+        createdAt:
+            expose: true
+            type: DateTime
+            groups: [Detailed]
+        updatedAt:
+            expose: true
+            type: DateTime
+            groups: [Detailed]
+    relations:
```

Comment:
I had similar issue, during work on [CustomerApi](https://github.com/Sylius/Sylius/pull/4037). We had to [resign with this approach](https://github.com/Sylius/Sylius/pull/4037/files#diff-d679bb45ecc6803f2c102d90bb0c84c4L29) in favor of proper serialization groups. [Here you have a few thoughts about that(with related comments](https://github.com/Sylius/Sylius/pull/4037#issuecomment-178563508). You would have to update a serialization group . Or find a better solution :dancer: